api/catalogue/graphql.go

```diff
@@ -18,12 +17,11 @@ import (
 var schema string
 
 type gql struct {
-	schema   *graphql.Schema
-	connpool *pgxpool.Pool
-	dbschema *psql.Schema
+	schema *graphql.Schema
+	client psql.IndexClient
```

Comment:
Why not a pointer? I am somewhat confused when we want a pointer and when a value.